packages/react-native-gesture-handler/src/utils.ts

```diff
@@ -45,13 +45,6 @@ export function tagMessage(msg: string) {
   return `[react-native-gesture-handler] ${msg}`;
 }
 
-// Helper method to check whether Fabric is enabled, however global.nativeFabricUIManager
-// may not be initialized before the first render
-export function isFabric(): boolean {
-  // @ts-expect-error nativeFabricUIManager is not yet included in the RN types
-  return !!global?.nativeFabricUIManager;
-}
-
 export function isReact19() {
   return React.version.startsWith('19.');
 }
```

Comment:
I think all versions of React Native that we support require React 19 by now, no? This could be removed as well if that's the case (in a follow-up).